### PR TITLE
Update several files

### PR DESCRIPTION
**Pull request by Lzw104522773**
Update the punctuation format of several files.

Update the tp2 file to avoid installation errors and avoid garbled prompts.

### CODE REVIEW

**Comment by 11jo** [comment]
You add new files without explanation ? 

[I have writen you a message when you first send the translation.](https://github.com/The-Gate-Project/DarkHorizons/commit/82c85223cc8627823193180a937b4e92f6f513a3#commitcomment-135839038)

Your setup-ee.tra was not encoded to UTF8, I suppose if it is now you don't need to add these new setup.tra or modify the tp2 for schinese language.

It's always possible that something goes wrong but if it work for other mod it should work here.

File: DarkHorizons/Language/schinese/setup-osx.tra [hunk 1]
```diff
@@ -0,0 +1,3846 @@
...
+防御等级：5
+重量：40~
+@17   = ~由精灵制造，这件锁子甲份量轻，弹性好。不象大多数锁子甲，精灵锁子甲不会限制魔法使用者练习他们的手艺。
+
+数据资料：
+
+防御等级：2
+重量：10
+~ 
+@18   = ~皮环甲~
```
Comment (Roberciiik) [inline DarkHorizons/Language/schinese/setup-osx.tra:45]: @Lzw104522773 It makes no sense to duplicate all of these strings from the original setup.tra and setup-ee.tra file. Instead, you should put here only those UTF-8 encoded texts that are required to be printed correctly during the installation. So in the tp2 file you can load setup.tra file first and then load your `setup-%OS%` version. All files are loaded one by one so the last one should overwrite the console texts.

**Comment by Lzw104522773** [comment]
I have reupload setup-%WEIDU_OS%.tra. You can check them again. Thank you.

**Comment by Lzw104522773** [comment]
> You add new files without explanation ?
> 
> [I have writen you a message when you first send the translation.](https://github.com/The-Gate-Project/DarkHorizons/commit/82c85223cc8627823193180a937b4e92f6f513a3#commitcomment-135839038)
> 
> Your setup-ee.tra was not encoded to UTF8, I suppose if it is now you don't need to add these new setup.tra or modify the tp2 for schinese language.
> 
> It's always possible that something goes wrong but if it work for other mod it should work here.

setup-osx.tra,setup-unix.tra,setup-win32.tra, It can avoid garbled installation prompts.

**Comment by 11jo** [comment]
@Roberciiik, 

May I leave this changes to you care, I really don't understand it and it seem a little to much for prompt. 

@Lzw104522773 
Since they are in UTF8 like setup-ee.tra, why loading the setup-ee.tra is not enough ?
Most languages don't use accent to avoid "garbled" prompt. I guess something is different in chinese.


But to clarify, I am not against if it's really necessary.

**Comment by Lzw104522773** [comment]
OK I see.

**Pull request closed without merging**